packages/sdks/src/blocks/img/img.lite.tsx

```diff
@@ -18,6 +18,7 @@ export interface ImgProps {
   imgSrc?: string; // TODO(misko): I think this is unused
   image?: string;
   altText?: string;
+  title?: string;
```

Comment:
this is the raw `img` component. it's different from `image`. I don't think you actually want to add it here? If you did, then:

- you need to update its componentInfo file
- you need to update the `Raw:Img` in gen1 react as well

Reply:
removed the change in this file